CONTRIBUTING.md

```diff
@@ -2,74 +2,68 @@
+3. **Treatment** - Provide suggestions on how to triage an issue.
 
-### Contribution Rewards
+### Asking for General Help
 
-Every month we allocate ~5,000 USD (ether fluctuates) to a reward pool (that we call "RewardDAO"). In order to be eligible all you need to do is complete a task with a given weight. At the end of the milestone (which can span in some cases span multiple months), we divide our RewardDAO budget evenly based on total points of contributions. Regular Reward members of the repository are not eligible for the RewardDAO so don't worry about us taking all those delicious points!
+Please feel free to direct questions or requests for general help in the [Aragon Planning riot channel](https://riot.im/app/#/room/#aragon-planning:matrix.org).
 
-Currently payouts are handled manually using a multi-sig however in the future all of this should be automated through the software we're building.
+## Pull Requests
 
-#### Regular Reward Members
+There are two fundamental components of the Pull Request process: one technical, and one process oriented. The technical component involves the specific details of setting up your local environment so that you can make the actual changes. This is where we will start.
 
-The following "Regular Reward Members" (aka full-time members) of the project, and are not eligible for the RewardDAO so don't worry about us taking all those delicious points!
-* quazia
-* stellarmagnet
-* rkzel
+### Environment Setup
 
-### Project Management
+We still need to document a comprehensive wiki on setting up the dev environment (want to help?), but this is what we have so far:
 
-We use Zenhub's Github Chrome Extension for project management, so [please install it](https://chrome.google.com/webstore/detail/zenhub-for-github/ogcgkffhplmphkaahpmffcafajaocjbd?hl=en-US). The only way you can see the points is with this extension.
+### Main Aragon documents/resources
 
-After installing the extension:
-* Navigate to the "Zenhub" tab within Github
-* Self-assign yourself unclaimed issues in the "Backlog" - ideally ones that have points allocated to them (we may be a bit behind in doing all of the planning poker - we will have another session soon).
+* https://wiki.aragon.one/dev/
+* https://github.com/aragon/aragonOS/blob/docs/docs/aragonOS.md
+* https://github.com/aragon/hack
+* https://gist.github.com/izqui/2a02c337aafa225c403de9c988cb78a0
+* https://github.com/aragon/aragon.js/blob/master/docs/APP.md
+* https://github.com/aragon/aragon.js/blob/master/docs/WRAPPER.md
 
+The WIP gist is probably the most helpful but also the most incomplete. If you follow the instructions, you'll run into some issues with some assumptions made about the APM permissions. As far as design, the app.md and wrapper.md are the most helpful for the front end, and the hack.md gives the best overview. The aragonOS.md details the solidity code and how the core of their system works.

```

Comment:
It might be good to suggest they ask in the chat if they need any help getting started.

Reply:
There's actually a link to the riot chat on https://github.com/Giveth/planning-app/pull/191/files#diff-6a3371457528722a734f3c51d9238c13R35 and in the README.